README.md

```diff
@@ -16,6 +16,7 @@ Using [Composer][composer]:
 
 ```
 composer require muffin/footprint
+
```

Comment:
Unnecessary whitespace change.

Reply:
Sorry was more of an accident than by intent.